### PR TITLE
release: 4.8.0

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR is to generate a new release of `code-server` at `4.8.0`

## Screenshot

image

## TODOs

Follow "Publishing a release" steps in `ci/README.md`

- [x] update `CHANGELOG.md`
- [ ] manually run "Draft release" workflow after merging this PR
- [ ] merge PR opened in [code-server-aur](https://github.com/coder/code-server-aur)

### CODE REVIEW

**Comment by jsjoeio** [comment]
Possible upstream issue with Helm: https://github.com/Azure/setup-helm/issues/99

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,29 @@ Code v99.99.999
...
+
+### Changed
+
+- Updated Code to 1.72.1
+
+### Fixed
+
+- Fixed bug in code-server (run with Docker) where PATH was always overwritten
+- Enalbed `BROWSER` environment variable
+- Patched `asExternalUri` to work so now extensions run inside code-server can use it
```
Comment (code-asher) [inline CHANGELOG.md:44]: Not sure on this but I think we need the `-ing` form of the verb here:
```suggestion
- Patched `asExternalUri` to work so now extensions running inside code-server can use it
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,29 @@ Code v99.99.999
...
+## [4.8.0](https://github.com/coder/code-server/releases/tag/v4.8.0) - 2022-10-24
+
+Code v1.72.1
+
+### Added
+
+- Support for the Ports panel which leverages code-server's built-in proxy. It
+  also uses `VSCODE_PROXY_URI` where `{{port}}` is replace when forwarding a port.
+  Example: `VSCODE_PROXY_URI=https://{{port}}.kyle.dev` would forward an
+  application running on localhost:3000 to https://3000.kyle.dev
```
Comment (code-asher) [inline CHANGELOG.md:32]: Love the example!  Wonderful use of the changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,29 @@ Code v99.99.999
...
+- support for `--disable-workspace-trust` CLI flag
+- support for `--goto` flag to open file @ line:column
+
+### Changed
+
+- Updated Code to 1.72.1
+
+### Fixed
+
+- Fixed bug in code-server (run with Docker) where PATH was always overwritten
```
Comment (code-asher) [inline CHANGELOG.md:42]: If I am thinking of the right thing this still happens in our default Debian-based image but now we should have Ubuntu images where this is not a problem.  So maybe we should move to features and say something like:

- Add Ubuntu-based images for Docker releases. If you run into issues with PATH being overwritten in Docker please try the Ubuntu image as this is a problem in the Debian base image.

File: patches/telemetry.diff [hunk 4]
```diff
@@ -3,7 +3,7 @@ Add support for telemetry endpoint
 To test:
 1. Create a RequestBin - https://requestbin.io/
 2. Run code-server with `CS_TELEMETRY_URL` set: 
-  i.e. `CS_TELEMETRY_URL="https://requestbin.io/1ebub9z1" ./code-server-4.7.1-macos-amd64/bin/code-server`
+  i.e. `CS_TELEMETRY_URL="https://requestbin.io/1ebub9z1" ./code-server-4.8.0-macos-amd64/bin/code-server`
```
[the remark this answers is not in the document]
Comment (jsjoeio) [reply patches/telemetry.diff:6]: Good call!

File: CHANGELOG.md (hunk 2 above)
Comment (jsjoeio) [reply CHANGELOG.md:32]: "Don’t let your friends dump git logs into changelogs." - applying what I learned from you!

File: CHANGELOG.md (hunk 3 above)
Comment (jsjoeio) [reply CHANGELOG.md:42]: Good call, i'll update

**Review by code-asher — APPROVED** [review]

**Pull request merged**